.circleci/config.yml

```diff
@@ -31,9 +33,114 @@ workflows:
+            wget https://storage.googleapis.com/dart-archive/channels/stable/release/latest/linux_packages/dart_2.14.4-1_amd64.deb
+            sudo dpkg -i dart_2.14.4-1_amd64.deb
+      - run:
+          name: Install FVM
+          command: |
+            dart pub global activate fvm
+            echo 'export PATH=$HOME/.pub-cache/bin:$PATH' >> $BASH_ENV
+      - restore_cache:
+          keys:
+            - flutter-3-{{ checksum ".fvm/fvm_config.json" }}-{{ arch }}
+      - run:
+          name: Install Flutter
+          command: |
+            fvm --verbose install
+      - save_cache:
+          key: flutter-3-{{ checksum ".fvm/fvm_config.json" }}-{{ arch }}
+          paths:
+            - .fvm
+            - ~/fvm/
+      - run:
+          name: Configure Flutter
+          command: |
+            fvm --verbose flutter config --no-analytics
+      - restore_cache:
+          keys:
+            - pub-2-{{ checksum "pubspec.lock" }}-{{ arch }}
+      - run:
+          name: Install Flutter Packages
+          command: |
+            fvm flutter pub get
+            fvm flutter precache --android
+      - save_cache:
+          key: pub-2-{{ checksum "pubspec.lock" }}-{{ arch }}
+          paths:
+            - .dart_tool
+      - run:
+          name: Build
+          command: |
+            fvm --verbose flutter build apk --release -t lib/main_prod.dart
+      - store_artifacts:
+          path: build/app/outputs/flutter-apk/
+
+  frontend-ios-build:
+    macos:
+      xcode: 13.2.0
+    working_directory: ~/project/frontend
+    steps:
+      - checkout:
+          path: ~/project
+      - run:
+          name: Install Standalone FVM (with Dart)
+          command: |
+            brew tap leoafarias/fvm
+            brew install fvm
+      - restore_cache:
+          keys:
+            - flutter-3-{{ checksum ".fvm/fvm_config.json" }}-{{ arch }}
+      - run:
+          name: Install Flutter
+          command: |
+            fvm --verbose install
+      - save_cache:
+          key: flutter-3-{{ checksum ".fvm/fvm_config.json" }}-{{ arch }}
+          paths:
+            - .fvm
+            - ~/fvm/
+      - run:
+          name: Configure Flutter
+          command: |
+            fvm --verbose flutter config --no-analytics
+      - restore_cache:
+          keys:
+            - pub-2-{{ checksum "pubspec.lock" }}-{{ arch }}
+      - run:
+          name: Install Flutter Packages
+          command: |
+            fvm flutter pub get
+            fvm flutter precache --ios
+      - save_cache:
+          key: pub-2-{{ checksum "pubspec.lock" }}-{{ arch }}
+          paths:
+            - .dart_tool
+      - run:
+          name: Update Pods
+          command: |
+            cd ios
+            pod update # https://github.com/m0nac0/flutter-maplibre-gl/pull/9
+            pod install
+      - run:
+          name: Build
+          command: |
+            fvm --verbose flutter build ios --no-codesign --release -t lib/main_prod.dart
```

Comment:
Why don't you save the .ipa as well?

Reply:
No IPA is build here, also just an ipa is quite useless if it is not signed with an ad-hoc certificate.